### ISSUE TITLE
Typo in b_hash_keys_to_list/2 predicate clause

### ISSUE THREAD

**Issue by pmoura** [vscosta/yap-6.3#47]
Missing `|` in the following clause of `b_hash_keys_to_list/2` predicate:

https://github.com/vscosta/yap-6.3/blob/61adeeeb5c8d9c93786528e8d1f4bbd4652839a7/library/bhash.yap#L302-L304

It should be:

```logtalk
b_hash_keys_to_list(hash(Keys, _, _, _, _, _), LKeys) :- 
     Keys =.. [_| LKs], 
     mklistels(LKs, LKeys). 
```

Found by the Logtalk linter.

**Comment by vscosta** [comment]
thanks!

**Comment by pmoura** [comment]
Fixed in e894209836a5dd4a7c54ae0d3e2b8a29e0673345.
